README.md

```diff
@@ -1,2 +1,114 @@
-# tw-academy-s9
-This is homework for session 9
+# Sample Style Guide
+## Technical Writing Academy - May 2021
+------
+## Table of Contents
+Bold
+
+Italics
+
+Fixed Font
+
+Abbreviations
+
+Capitalization
+
+Acronyms
+
+Commas in Lists
+
+Numbers
+
+Contractions
+
+Apostrophes
+
+## **Bold**
+Use bold to highlight button names, fields names, menus, radio buttons, or checkboxes. For example:
+
+● Click the Create button to start the process.
```

Comment:
Global comment: You can use an asterisk or hyphen to create the bullets. Markdown will format them as bullets. For example:
- This is bullet one
- This is bullet two

Reply:
(in the previous example, you will see a bullet list, but actually I wrote just a hyphen ( - ) at the start of each line, and GitHub formats it as a bullet)